backends-common/opensearch/pom.xml

```diff
@@ -88,7 +88,7 @@
         <dependency>
             <groupId>org.opensearch.client</groupId>
             <artifactId>opensearch-rest-client</artifactId>
-            <version>2.4.0</version>
+            <version>2.8.0</version>
```

Comment:
You forgot to revert this one too :)

Reply:
the test cases still pass with only upgrade opensearch-rest-client, so I still keep it
Do you think we should do both opensearch-rest-client, opensearch-java same time?

Reply:
Ok for keeping it. 

No the goal of upgrading opensearch-java is that it might allow us to drop opensearch-rest-client dependency for good :)